test/BUILD

```diff
@@ -46,6 +46,7 @@ mypy_test(
 mypy_test(
     name = "broken_mypy_test",
     deps = [":broken_mypy_typings"],
+    tags = ["manual"],
```

Comment:
I think this was just absent before, these two tests are meant to fail which prevents `bazel test //...` in this repo